index.html

```diff
@@ -44,5 +43,140 @@ <h3 class="small-txt">LET'S CONNECT</h3>
+    <h3 class="tag">Back End Dev</h3>
+    <div class="dot-separator"></div>
+    <h3 class="tag">2015</h3> </div>
+     <div class="card-txt"><p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p></div>
+
+     <div class="prog">
+      <ul>
+        <li><button type="button">html</button></li> 
+        <li><button type="button">css</button></li>
+        <li><button type="button">javaScript</button></li>
+     </ul>
+      </div>
+      <div class="project-check"> <button type="button">See Project</button> </div> 
+   </div>
+
+   <div class="card-4">
+   <img src="Images/Snapshoot-Portfolio-image-4.png" alt="" class="card__image">
+
+   <h2 class="big-txt">Multi-Post Stories</h2>
+   <div class="tag-group">
+    <h3 class="tag-Canopy">CANOPY</h3>
+    <div class="dot-separator"></div>
+    <h3 class="tag">Back End Dev</h3>
+    <div class="dot-separator"></div>
+    <h3 class="tag">2015</h3> </div>
+    <div class="card-txt"><p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p></div>
+
+    <div class="prog">
+      <ul>
+        <li><button type="button">html</button></li> 
+        <li><button type="button">css</button></li> 
+        <li><button type="button">javaScript</button></li>
+     </ul>
+      </div>
+      <div class="project-check"> <button type="button">See Project</button> </div> 
+  </div>
+
+   </section>
+
+  <header class="header-2">
+    <div class="main-headline-txt">
+
+       <h2 class="glad-txt">About<br>Myself</h2>
+   
+      <p class="paragraph-txt">
+        I am a beginner student in the Microverse Program, this is my first week!
+        I'm learning right now how to implement flexbox properties in CSS. 
+        This is my first project using flexbox and setting up a mobile version skeleton
+      </p>
+
+      <div class="connect">
+        <h3 class="small-txt">LET'S CONNECT</h3>
+   </div>
+   <div class="media-Images">
+      <ul>
+        <li><a href="#"> <img src="Images/GitHub-Desktop-icon.svg" alt="Github Icon"></a></li>
+        <li><a href="#"> <img src="Images/Linkedin-icon.svg"   alt="Linkedin Icon"></a></li>
+        <li><a href="#"> <img src="Images/Peace-Emoji-Icon.svg"  alt="Peace Icon"></a></li>
+        <li><a href="#"> <img src="Images/Twitter-icon.svg"   alt="Twitter Icon"></a></li>
+        <li><a href="#"> <img src="Images/Elegant-M-Icon.svg"   alt="Micro Icon"></a></li>
+      </ul>    
+       </div>
+       <div class="project-check"> <button type="button">Get my resume</button> </div>
+   </div>
+  </header>
+
+  <section class="Programs-section">
+    <div class="list-container"> 
+
+      <ul><li class="lang-label"> <h3>Languages</h3> <img src="Images/Arrow_down.svg" alt="Arrow_down"></li>
+        <li><img src="Images/JavaScript-bar.png" alt="javaScript"></li> 
+        <li><img src="Images/HTML-bar.png" alt="HTML"></li>
+        <li><img src="Images/CSS-bar.png" alt="CSS"></li>
```

Comment:
Kindly, I recommend that all these languages are HTML code and only use the images for the logo.